### ISSUE TITLE
Improve Pop-Up Dialog when Exiting RSP Session

### ISSUE THREAD

**Issue by wmwv** [rubin-dp0/Support#8]
**Is your feature request related to a problem? Please describe.**
I clicked "Exit Without Saving and Log Out" and got a pop-up window of

"""
Redirecting to landing page

Please wait until you are redirected back to the landing page.
[OK]
"""

It was confusing.  Do I click "OK"?  Do I have to wait to click "OK" until I get back to the landing page?  Why does the system need any further interaction from me at all?  Is there something client-side that still needs to happen after I select "Exit Without Saving and Log Out"?  What happens if I just close my browser, or sleep my computing immediately?

**Describe the solution you'd like**
1. Ideally I think there shouldn't be any need for a pop-up dialog given that I selected the "Exit Without Saving and Log Out" option.  If I selected "Exit Without Saving", then I understand the dialog box, but with the "Exit Without Saving and Log Out" option, I don't expect a dialog box.
2. But perhaps the pop-up dialog I actually get when click either "Exit Without Saving" or "Exit Without Saving and Log Out" is one that asks "Do you want to exit without saving?  [YES]/[NO]"

**Describe alternatives you've considered**
3. A pop-up dialog box that more clearly explains why I need to wait.  I understand that in this version of the RSP there may indeed be something client-side that is awkward to take out right now.

**Additional context**

<img width="415" alt="Screen Shot 2021-07-08 at 09 00 41 " src="https://user-images.githubusercontent.com/3201257/124957598-198f2280-dfce-11eb-9590-b64aa4557044.png">

**URL**
https://data.lsst.cloud/nb/user/wmwv/lab?

**Comment by SimonKrughoff** [comment]
@wmwv thank you for the suggestion.  To be honest, I'm not sure why there is a dialog at all.  People are certainly used to being redirected when they click the logout button for a webpage.  I'll make a ticket to explore further.

**Comment by frossie** [comment]
FWIW the reason there is a popup is because we want to finish shutting down your container before you are redirected; and the reason for _that_ is if you ignore this warning (or if we redirected you right away) and go back to the landing page yourself and then ask for a new notebook session while your previous container is still shutting down (which the nimble fingered can do), then you get messages that seemed to be confusing to users (like "your container is still stopping" and the like). 

tl;dr: it's there to prevent even more confusing behaviour

When we revisit the JL environment in further planned work we will add it to the list of things to look at to see we have any better ideas.

**Comment by frossie** [comment]
@wmwv try using the daily 8/24 image and tell us whether you find the change more, less or equally confusing
